sample-code/pmd-example-rules/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This is a sample POM file allowing this module to be compiled and bundled into a JAR via maven.
+If you have Maven properly configured on your machine, you can run `mvn clean package` to build
+a JAR containing everything needed to register and run a custom rule.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
I would think that this sample project was locally built with this pom file and the build works.  I just can't verify the semantic validity of the pom file is all am saying